cloud/ec2_fleet.go

```diff
@@ -144,6 +144,7 @@ func (m *ec2FleetManager) SpawnHost(ctx context.Context, h *host.Host) (*host.Ho
 		"host_provider": h.Distro.Provider,
 		"distro":        h.Distro.Id,
 	})
+	h.RevokeGithubTokens(ctx)
```

Comment:
Why is the token revoked here? It seems like the token wouldn't even exist yet by this point because `PopulateGitHubTokens` hasn't been called yet.